lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -162,6 +152,19 @@ def verify_real_user_profile_link(self):
             "Profile link should point to real user",
         )
 
+    def masquerade_as_group_member(self, partition_id, group_id):
```

Comment:
I find it confusing that this has the same name as the method it is wrapping. Maybe call it ``ensure_masquerade_as_group_member`` or something similar to indicate that it only works for 200s.